spec/integration/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -34,6 +34,16 @@ spec:
           enum:
             - docker
           default: docker
+        type:
+          description: >
+            "Elastic Agent image to be used for testing. Setting default will be used the
+            same Elastic Agent image as the stack."
+          type: string
+          enum:
+            - default
+            - complete
+            - minimal
```

Comment:
Complete would mean to use `elastic-agent-complete` image for docker.
Minimal would mean to use `elastic-agent` image for docker.
Default the same image as the Elastic Agent from the stack.